docs/synthetics/user-journey-scripts.md

```diff
@@ -107,3 +107,30 @@ The logs will also contain the Sematext API response indicating the sync failure
 ```
 {"success":false,"message":"invalid monitor id 271"}
 ```
+
+## Storing your User Journey script credentials securely
+Earlier on we mentioned that Browser monitors can be used to interact with a website. This interaction often includes logging in, so that one may access the features available only to registered users. Logging in is usually performed by passing certain credentials to a form on the page and then submitting it. Since these credentials must be supplied inside of the User Journey script, we understand that users might be concerned with their security. By using our **Add Sensitive Data** functionality, your credentials will be securely stored in the AWS Secrets Manager. That means they won't be visible in a plaintext format inside of scripts, instead being replaced by a placeholder value which you specify. These placeholders are replaced with the actual values only during the User Journey script's execution.
```

Comment:
I'd link to the secrets section here instead of just emphasizing.

Reply:
This is the secrets section